README.md

```diff
@@ -195,13 +195,50 @@ prebuilt_gsx/
 
 Run build as usual with `ninja`.
 
-## Building with prebuilts Android graphics
+## Building with prebuilt graphics and Android guest
 
-Prior to running moulin, you need to place android graphics prebuilts
-archive `rcar-prebuilts-graphics-xt-doma.tar.gz` in the same directory
-as yaml file.
+### Preparation
 
-Use `--GRAPHICS binaries` command line option for moulin.
+You need to prepare prebuilt graphic binaries before building. Since Android
+guest is supported only in the virtio-based build, we use prebuilt graphic
+binaries without virtualization.
+
+1. Create a folder for the prebuilt binaries
+`<directory_with_yaml>/../prebuilt_gsx/domd/`. You may use any other folder,
+but it should be specified inside the `prod-devel-rcar-virtio.yaml` variable
+`XT_PREBUILT_GSX_DIR`.
+
+2. Download the "R-Car Gen3 Evaluation Software Package for Linux for Yocto v5.9.0"
+from Renesas site:
+https://www.renesas.com/us/en/application/automotive/r-car-h3-m3-h2-m2-e2-documents-software
+(registration may be required).
+
+Two files are required:
+- [R-Car_Gen3_Series_Evaluation_Software_Package_for_Linux-20220121.zip](https://www.renesas.com/en/document/swo/r-cargen3seriesevaluationsoftwarepackageforlinux-20220121?r=1245356)
+- [R-Car_Gen3_Series_Evaluation_Software_Package_of_Linux_Drivers-20220121.zip](https://www.renesas.com/en/document/swo/r-cargen3seriesevaluationsoftwarepackageoflinuxdrivers-20220121?r=1245356)
+
+3. Extract the required drivers from the downloaded pack of archives.
+
+- From `R-Car_Gen3_Series_Evaluation_Software_Package_of_Linux_Drivers-20220121.zip`
+get `RCH3G001L5101ZDO_4_2_1.zip`. From that last archive,
+get `GSX_KM_H3.tar.bz2`, and put it into the folder `prebuilt_gsx/domd/`
+created in step 1.
+
+- From `R-Car_Gen3_Series_Evaluation_Software_Package_for_Linux-20220121.zip`
+get `INFRTM8RC7795ZG300Q10JPL3E_4_2_1.zip`. From that last archive, get
+`INF_r8a77951_linux_gsx_binaries_gles.tar.bz2`.
+
+Rename `INF_r8a77951_linux_gsx_binaries_gles.tar.bz2` to
+`r8a77951_linux_gsx_binaries_gles.tar.bz2`, and put it into the folder
+`prebuilt_gsx/domd/`  created in step 1.
+
```

Comment:
It would be nice to provide listing of the `prebuilt_gsx/domd/`  directory, so user can verify their work. 

(Much better approach is to automate all this unpacking right in the DDK recipe, but this is for latter)

Reply:
I have added an example with the prepared files.